.rubocop.yml

```diff
@@ -2,7 +2,8 @@ require:
   - rubocop-rails
 
 AllCops:
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 2.5
+  NewCops: disable
```

Comment:
New cops can be falsely detected, so I thought it wouldn't be necessary to actively enable them. The current unconfigured state is pending and the new cop remains disabled.
By setting it explicitly, the warning that prompts you to set the new cop will not be displayed.